### PR TITLE
Add More Attributes to the device page

### PR DESCRIPTION
**Pull request by karls4**

### CODE REVIEW

File: modern/src/DevicePage.js [hunk 1]
```diff
@@ -88,6 +96,33 @@ const DevicePage = () => {
               label={t('deviceIdentifier')}
               variant='filled' />
           }
+          {(!id || device) &&
```
Comment (tananaev) [inline modern/src/DevicePage.js:99]: We should probably move this check up, so it's not repeated so many times.

File: modern/src/DevicePage.js [hunk 2]
```diff
@@ -38,10 +40,16 @@ const DevicePage = () => {
     });
   }, [id]);
 
+
+
```
Comment (tananaev) [inline modern/src/DevicePage.js:44]: Please remove extra lines.

File: modern/src/DevicePage.js [hunk 3]
```diff
@@ -71,22 +76,47 @@ const DevicePage = () => {
...
-          }
-          {(!id || device) &&
-            <TextField
-              margin='normal'
-              fullWidth
-              defaultValue={device && device.uniqueId}
-              onChange={(event) => setUniqueId(event.target.value)}
-              label={t('deviceIdentifier')}
-              variant='filled' />
+            <div>
```
Comment (tananaev) [inline modern/src/DevicePage.js:79]: Is there a way to avoid `div`? Can we use fragment here?

Comment (karls4) [reply modern/src/DevicePage.js]: Sorry I was unaware of fragment.

File: modern/src/DevicePage.js [hunk 4]
```diff
@@ -71,22 +76,47 @@ const DevicePage = () => {
...
-          }
-          {(!id || device) &&
-            <TextField
-              margin='normal'
-              fullWidth
-              defaultValue={device && device.uniqueId}
-              onChange={(event) => setUniqueId(event.target.value)}
-              label={t('deviceIdentifier')}
-              variant='filled' />
+            <React.Fragment>
```
Comment (tananaev) [inline modern/src/DevicePage.js:79]: Please use empty tag instead for consistency with other code.

**Comment by karls4** [comment]
image

See attached. Its plain and simple at the moment

**Comment by tananaev** [comment]
Also already implemented.

**Pull request closed without merging**